manifests/kiali-upstream/1.90.0/manifests/kiali.v1.90.0.clusterserviceversion.yaml

```diff
@@ -521,14 +521,15 @@ spec:
           - projects
           verbs:
           - get
-        - apiGroups: ["route.openshift.io"]
-          resources:
-          - routes
-          verbs:
-          - get
         - apiGroups: ["authentication.k8s.io"]
           resources:
           - tokenreviews
           verbs:
-          - create
```

Comment:
did this "create" verb get deleted by accident? I see tokenreviews still has "create" in other places and there seems to be an empty "verbs" section in here now.